runtest.js

```diff
@@ -66,5 +70,30 @@ process.env.__CONFIGURATION = JSON.stringify(config);
 // Setting real ARGV
 process.argv = argv;
 
-// Execute Jest. Specific target optional.
-shell.exec(`npm run test ${testPath} ${runInBand}`);
+const isLocal = config.site === "local";
+const infoURL = (isLocal ? "http://" : "https://") + config.site + "my.good-loop.com/build/gitlog.txt";
+
+if (!yargv.skipProdTest) {
+	// Check tests are not running on production
+	console.log("Checking gitlog for host type...");
```

Comment:
Maybe an extra comment about the external assumptions - e g.

// Assume gitstamp.sh was run when the web-app was built, creating gitlog.txt